CICDTemplate/README.md

```diff
@@ -0,0 +1,38 @@
+# Setup
```

Comment:
Have you taken a look at #76? What is your idea about keeping these CICD templates? Should we put all of them in the same folder or having separate child folders inside `CICDTemplate`?

Instead of having `CICDTemplate`, how about `.bitrisetemplate` and `.githubtemplate` in the root folder 🤔 ?

Reply:
@luongvo Yeah. I did 😅 I put it in `.bitrise` but it seems to be like the configuration folder instead of templates. So defining the folder with `Template` like `RxTemplate` or `JavaTemplate` could make more sense,

https://github.com/nimblehq/android-templates/pull/77#issuecomment-838127735

Reply:
I think @luongvo meant that he'd also prefer to do this: https://github.com/nimblehq/android-templates/pull/76 `.githubtemplate` instead of creating a normal directory. I'm waiting for you guys to conclude, will follow the convention to update my PR if needed.

Reply:
Yup @sleepylee, @hoangnguyen92dn creating CICDTemplate folder makes me feel like we will have the third template project name as CICDTemplate, while having RxJavaTemplate and CoroutineTemplate.

Reply:
@luongvo Hmm, make sense 🤔 I think all the `yml` (Bitrise, Github Actions, ...) should be located in the same place, so I would prefer to have the folder structure like this:
- `.cicdtemplate`
  - `.bitrise`
  - `.github`
  - ...

What do you guys think? @sleepylee @luongvo

Reply:
that would work too. as long as it's easy for you later to templating with a command
i.e `cp .cicdtemplate/.bitrise .bitrise` to the actual new project, it's fine

Reply:
@hoangnguyen92dn works for me 👌